### ISSUE TITLE
Clarification on Parallelisation of batches

### ISSUE THREAD

**Issue by philastrophist** [joshspeagle/dynesty#223]
Hi, 

I'm trying to use dynesty to map a highly mulimodal, computationally-intensive posterior (a gaussian mixture) and I'd like to know how to parallelise. 
I understand that we can use a multiprocess/MPI pool and I do it that way atm, but you also mention that we can add batches of runs together post-hoc. 
I can't find any references which talk about what effect that has on the posterior/evidence. Does it just increase resolution or do separate runs extend exploration (this seems doubtful). 

Basically, are run-batch-combinations equivalent to just running longer? 

I'm just trying to work out if it would be beneficial for me to break up into many batches.

**Comment by joshspeagle** [comment]
It essentially just increases resolution. Combining N runs with K live points is equivalent to running a single run with NK live points.

**Comment by philastrophist** [comment]
Right, and I'm guessing there is no reason to expect a combination to map modes that an extended run did not find?

**Comment by joshspeagle** [comment]
Not as much, especially since you're more likely to miss modes with runs that use fewer live points initially.

**Comment by philastrophist** [comment]
ok, that's what I thought, thanks!
